src/SUMMARY.md

```diff
@@ -71,6 +71,7 @@
 * [Variables](development/variables.md)
 * [Accessing your site](development/access-site.md)
 * [Headers](development/headers.md)
+* [Email](development/email.md)
```

Comment:
I dont understand why this has moved... but then I dont think dev/admin should be separate sections either.

Reply:
It's old location was under administration/web, which makes little sense as it has nothing to do with the web UI.  But I get asked about it while people are developing all the time.